DataFormats/Portable/BuildFile.xml

```diff
@@ -1,4 +1,4 @@
 <use name="alpaka"/>
 <use name="rootcling"/>
 <use name="FWCore/Utilities/" source_only="1"/>
-<use name="HeterogeneousCore/AlpakaInterface" source_only="1"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
```

Comment:
This because `source_only="1"` does not propagate the module's external dependencies.